acceptance/acceptance_test.go

```diff
@@ -40,6 +41,8 @@ func TestBuilder(t *testing.T) {
 			},
 			{
 				Name:    "java-boot",
+				Repo:    "https://github.com/ericbottard/fats",
```

Comment:
Remove these two lines once https://github.com/projectriff/fats/pull/183 is merged